__tests__/__snapshots__/RNColorPlalette-test.js.snap

```diff
@@ -0,0 +1,307 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
Do we need to commit these snapshots? Just curious.

Reply:
removed.